integration_test/scripts/publish-images.sh

```diff
@@ -1,14 +1,37 @@
 #!/bin/bash
-# imageProject publishFlag
+# imageProject publishFlag repositoryString
+
+accessToken=$ACCESS_TOKEN
+repositoryString=$3
+IFS='_' read -r -a repoArray <<< "$repositoryString"
```

Comment:
Could you add a comment explaining the format of the repo string and how it changes in certain circumstances?

Reply:
repositoryString contains "githubUser_githubRepo" if repository is stored by user  and just "githubRepo" if stored by organization https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values